taskvine/src/manager/vine_file.c

```diff
@@ -143,6 +143,20 @@ struct vine_file *vine_file_clone(struct vine_file *f)
 	return f;
 }
 
+/* Make a URL reference to a file source*/
+
+char *vine_file_make_file_url(const char *source)
+{
+
+	char *abs_path = path_getcwd();
+
+	char *result = string_format("file:%s/%s", abs_path, source);
```

Comment:
Missing a '/'? (file://)

Reply:
The resulting url should be `file://{the absolute path} ->  file:///some_path`. url start with `spec://`.  So it will end up with three slashes, two required for the spec, and one from the absolute path.